Documentation/ContentObjects/Extbaseplugin/Index.rst

```diff
@@ -62,6 +62,42 @@ Example
     page.10.extensionName = MyExtension
     page.10.pluginName = MyPlugin
 
+..  _cobj-extbaseplugin-examples-fluid:
+
+Example: Display an Extbase plugin in a Fluid template
+======================================================
+
+It is possible to display an Extbase plugin in within Fluid using the
+:ref:`CObject ViewHelper <f:cObject> <t3viewhelper:typo3-fluid-cobject>`:
+
+..  literalinclude:: _CodeSnippets/_SomeTemplate.html
+    :caption: EXT:myExtension/Configuration/TypoScript/setup.typoscript
```

Comment:
Do we not use "my_extension" instead of "myExtension" as extension key throughout the docs? Also in other places in the PR used.